freqtrade/constants.py

```diff
@@ -56,7 +56,7 @@
 
 ENV_VAR_PREFIX = 'FREQTRADE__'
 
-NON_OPEN_EXCHANGE_STATES = ('cancelled', 'canceled', 'closed', 'expired')
+NON_OPEN_EXCHANGE_STATES = ('cancelled', 'canceled', 'expired')
```

Comment:
This is introducing a bug, please revert.

Please avoid changes to areas of the bot you don't understand (or without analyzing it's full impact).